packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -53,13 +53,15 @@ export default declare((api, opts) => {
   const {
     jsxPragma = "React.createElement",
     jsxPragmaFrag = "React.Fragment",
-    allowNamespaces = false,
```

Comment:
This can use:
```suggestion
    allowNamespaces = Boolean(process.env.BABEL_8_BREAKING),
```

Reply:
No, because we don't support statically removing it when publishing.